_posts/en/newsletters/2021-06-02-newsletter.md

```diff
@@ -0,0 +1,144 @@
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20833][] is the first PR in an effort to implement
+  [mempool package acceptance][package mempool accept blog] in Bitcoin
+  Core. This change allows the `testmempoolaccept` RPC
+  to accept multiple transactions where later transactions may be
+  descended from earlier transactions. Future PRs may enable testing L2 transaction
+  chains, submitting transaction packages directly to the mempool through RPCs
+  and communicating packages over the P2P network.
+
+- [Bitcoin Core #22017][] updates the code signing certificate used for
+  Windows releases after the previous certificate was revoked by its
+  issuer without them providing an explicit reason.  Several recent
+  releases of Bitcoin Core might be re-released with slightly different
+  version numbers so that their Windows binaries can use this
+  certificate.
+
+- [Bitcoin Core #18418][] increases the maximum number of UTXOs received
+  to the same address that will be spent simultaneously if the
+  `avoid_reuse` wallet flag is set.  The more outputs that are spent
+  together, the higher the fee might be relative to a wallet with
+  default flags but, also, the less likely it becomes that third parties
+  will be able to identify the user's later transactions.
+
+- [C-Lightning #4501][] adds [JSON schemas][] for the output of
+  roughly half of C-Lightning's current commands (with schemas for the
+  other half planned to be added in the future).  Output produced during
+  a run of C-Lightning's test suite is validated against the schemas to
+  ensure consistency.  The schemas are also used to automatically
+  generate C-Lightning's documentation about what output each command
+  produces.
+
+- [LND #5025][] adds basic support for using [signet][topic signet].  Of
+  the other LN implementations tracked by Optech, C-Lightning also has
+  support for signet (see [Newsletter #117][news117 cl4068]).
+
+- [LND #5155][] adds a configuration option to randomly select which
+  wallet UTXOs to spend in a transaction; this reduces UTXO
+  fragmentation in the wallet over time.  By contrast, the default coin
+  selection algorithm in LND spends higher value UTXOs before lower
+  value UTXOs; this minimizes fees in the short term but may result in
+  needing to pay higher fees in the future when all inputs near the size
+  of a transaction, or larger, have already been spent.
+
+- [BOLTs #672][] updates [BOLT2][] to allow nodes to negotiate a
+  `option_shutdown_anysegwit` option which, if set, allows LN closing
+  transactions to be able to pay any segwit script version, including
+  script types that don't yet have consensus meaning on the network,
+  such as addresses for [taproot][topic taproot].
+
+- [BOLTs #872][] updates [BOLT3][]'s use of [BIP69][] to specify in more
+  detail the sort order to use for commitment transaction inputs and
+  outputs.  One commentator points out that the use of BIP69 has so far
```

Comment:
why "one commentator"? seems inconsistent because [there have already been many mentions of Corallo](https://www.google.com/search?q=site%3Abitcoinops.org+%22matt+corallo%22)

Reply:
@ysangkok We almost never[1] mention names in the *notable changes* section.  The reason is that notable changes to large projects usually require the close cooperation of many participants---designers, implementers, reviewers, testers, maintainers, etc---and I worry it would be unfair and bad for morale if only a subset of those people received recognition for the whole group's effort.  In this particular case, several people who put quite a lot of debugging effort into identifying this as the cause of accidental channel force closures, while others came up with this simple solution and then implemented and tested it, and it would seem really unfair to me if the only name we mentioned out of all that work was Corallo's for a quick comment he made re-stating his objections to BIP69 (which are well known to other LN maintainers).

[1] Off hand, I'm only aware of one exception to this policy, see the last two sentences of https://bitcoinops.org/en/newsletters/2019/03/26/#bitcoin-core-10973